.discourse-compatibility

```diff
@@ -1,3 +1,4 @@
+< 3.4.0.beta4-dev: a53719ab8eb071459f215227421b3ea4987e5f87
```

Comment:
Technically covered by the below line, but I think this is correct, in case of the PR adding the below being reverted, etc.